src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodes.cs

```diff
@@ -1361,13 +1379,19 @@ protected override void OnExpandPopulateFolders(IList<TreeNode> currentChildren,
 					},
 					new NodeFilterProperty
 					{
-						Name = "DurabilityType",
+						Name = "Durability",
 						DisplayName = SR.FilterDurabilityType,
 						Type = NodeFilterPropertyDataType.Choice,
 						Description = SR.FilterDurabilityTypeDescription,
-						Choices = new string[] {
-							"SchemaAndData",
-							"SchemaOnly",
+						Choices = new NodeFilterPropertyChoice[] {
```

Comment:
Should we move this array instantiation to a separate method and just reference that to avoid having to change things in 4 different locations if choices are added or removed in the future?

Reply:
This is generated code. So repetitions don't matter much.